IdentityMicroService/Application/Services/AuthenticationService.cs

```diff
@@ -119,5 +127,45 @@ public async Task<bool> CreatePatientAsync(RegistrationUserDTO model)
 
             return true;
         }
+
+        public async Task<bool> SendEmailConfirmAsync(RegistrationUserDTO model, IUrlHelper url)
+        {
+            var user = await _userManager.FindByEmailAsync(model.Email);
+            if(user != null)
+            {
```

Comment:
You can avoid such a big nesting replacing this with if (user == null) {return false;} and then paste your code